apps/web/src/server/config/sso.js

```diff
@@ -0,0 +1,13 @@
+import { loadEnvironment } from '@pins/platform';
+
+loadEnvironment(process.env.NODE_ENV);
```

Comment:
isn't the idea that loadEnvironment is only used within apps/web/src/service/config/config.js, and then everything else imports from there?